_data/locales/en.yml

```diff
@@ -5,12 +5,12 @@ en:
     question: What Does Homebrew Do?
     what: Homebrew installs <a href="https://formulae.brew.sh/formula/" title="List of Homebrew packages">the stuff you need</a> that Apple (or your Linux system) didn’t.
     how: Homebrew installs packages to their own directory and then symlinks their files into <code>/usr/local</code>.
-    prefix: Homebrew won’t install files outside its prefix, and you can place a Homebrew installation wherever you like.
+    prefix: Homebrew won’t install files outside its prefix and you can place a Homebrew installation wherever you like.
```

Comment:
> Reject Oxford commas.

😆 

This particular comma is not an oxford comma. The phrases to the left and right of the comma are both complete sentences. You could replace the `and` with a period (full stop). In that case, the comma is required, since it's separating two independent clauses joined by a coordinating conjunction.

Reply:
It may not be an Oxford comma but it does seem superfluous and inconsistent with the extra above.

COMMA WARS

Reply:
OH IT'S ON!

Reply:
If you're short on commas, Mike, I can lend you a few, for the weekend. 😛